src/mgmt/srv.c

```diff
@@ -445,6 +441,12 @@ add_ownership_props(daos_prop_t **prop_out, char *owner, char *owner_grp)
 		entries++;
 	}
 
+	if (entries == 0) {
+		D_ERROR("No owner identities provided, aborting!\n");
+		rc = -DER_INVAL;
```

Comment:
Do we really want to treat this as an error? No owner feels pretty weird conceptually, but I don't know if it's a valid use case in the long term (i.e. when we support setting user/group ACEs). I suppose we can always change it.

Reply:
yes at the moment it should be an error because it would get populated on the go side if left as an empty parameter in the call. make sense?